src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -2884,13 +2887,23 @@ bool Creature::SetWalk(bool enable)
     return true;
 }
 
-bool Creature::SetDisableGravity(bool disable, bool packetOnly/*=false*/)
+bool Creature::SetDisableGravity(bool disable, bool packetOnly /*=false*/, bool updateAnimationTier /*= true*/)
 {
     //! It's possible only a packet is sent but moveflags are not updated
     //! Need more research on this
     if (!packetOnly && !Unit::SetDisableGravity(disable))
```

Comment:
Here, too.

(And I know it doesn't use it, but it's still good practice in case it ever gets changed.)